bin/server/cmd/root.go

```diff
@@ -162,6 +162,8 @@ type Main struct {
 	l *zap.Logger
 
 	server *server.Server
+	// serverCancel cancels any runner on the server.
```

Comment:
Would be better to comment the reason for introducing this (use the one in the PR description)

Reply:
Done